spec/controllers/subscriptions_controller_spec.rb

```diff
@@ -28,6 +28,55 @@
   end
 
   describe '#create' do
+    let(:subscription) do
```

Comment:
Do not these new `let` blocks belong inside the scope of the new `context` block, as they aren't used outside it?

Reply:
Ah, yes, I was going to add another block of tests, but they aren't necessary.  I'll move them inside.